GitUI/GitUICommands.cs

```diff
@@ -1920,14 +1920,14 @@ public IGitRemoteCommand CreateRemoteCommand()
         }
 
         /// <summary>
-        ///  Creates a new instance of <see cref="GitUICommands"/> for a git repository specified by <paramref name="module"/>.
+        /// Creates a new instance of <see cref="GitUICommands"/> for a git repository specified by <paramref name="module"/>.
```

Comment:
I've done this on purpose. The double space makes it easier to read comments. We use this convention in a few .NET repos, notably in dotnet/winforms.

Reply:
> The double space makes it easier to read comments.

This is pretty new - and done inconsistently in this PR (at least in `Plugins/GitUIPluginInterfaces/ServiceProviderExtensions.cs`). I do not have strong opinion on it.